dist/components/Blob.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from "react";
+
+import styled from "styled-components";
+
+import * as THREE from "three";
+import SimplexNoise from "simplex-noise";
+
+const Blob = () => {
+  useEffect(() => {
+    let $canvas = document.querySelector("#blob canvas"),
```

Comment:
we should probably use a ref versus querySelector? querySelector requires walking the DOM tree.